app/components/provider_interface/application_card_component.html.erb

```diff
@@ -4,6 +4,10 @@
     <%= govuk_link_to candidate_name, provider_interface_application_choice_path(application_choice) %>
     <div class="app-application-card__status">
       <%= render ProviderInterface::ApplicationStatusTagComponent.new(application_choice: application_choice) %>
+      <% if most_recent_note %>
+        <div class="govuk-!-margin-top-0 govuk-!-margin-bottom-0"></div>
```

Comment:
You probably took this from the prototype but the prototype isn’t perfectly clean. it would be better if this div wasn’t here.

Reply:
Fixed